app/models/manageiq/providers/inventory/persister/builder/infra_manager.rb

```diff
@@ -128,37 +128,11 @@ def storages
 
         def hosts
           add_properties(
-            :attributes_blacklist => %i[parent],
-            :delete_method        => :disconnect_inv
+            :attributes_blacklist   => %i[parent],
+            :delete_method          => :disconnect_inv,
+            :custom_reconnect_block => INVENTORY_RECONNECT_BLOCK
```

Comment:
where is `INVENTORY_RECONNECT_BLOCK` defined?

Reply:
It is in the shared module (linked in the description)